appengine/genetics/style.css

```diff
@@ -0,0 +1,171 @@
+#blockly {
+    position: fixed;
+    bottom: 10px;
+}
+
+#editor {
+    font-size: 16pt;
+    position: fixed !important;
```

Comment:
The CSS is largely based off of the pond-duck game. I'll have to take a closer look to see if its necessary.